packages/response-ops/feature_flags/is_feature_flag_set.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { featureFlags } from './feature_flags';
+
+export const isFeatureFlagSet = (feature: string) => {
+  return featureFlags.has(feature);
```

Comment:
I am skeptical about a scenario where we remove a feature flag from the `featureFlags` set but in the code, we forgot to remove the `isFeatureFlagSet` check. In this scenario, the function will always return `false`. I think it is fine as we should add tests that the feature is working as expected when we remove the feature flag.